docs/content/common-subs/kubestellar-apply-apache.md

```diff
@@ -0,0 +1,111 @@
+<!--kubestellar-apply-apache-start-->
+<span class="Space-Bd-BT">KUBESTELLAR</span>'s helm chart automatically creates a Workload Management
+Workspace (WMW) for you to store kubernetes workload descriptions and <span class="Space-Bd-BT">KUBESTELLAR</span> control objects in. The automatically created WMW is at `root:wmw1`.
+
+Create an EdgePlacement control object to direct where your workload runs using the 'location-group=edge' label selector. This label selector's value ensures your workload is directed to both clusters, as they were labeled with 'location-group=edge' when you issued the 'kubestellar prep-for-cluster' command above.
+
+In the `root:wmw1` workspace create the following `EdgePlacement` object: 
+```shell linenums="1" hl_lines="10 11 16 21"
```

Comment:
The display of line numbers here is inconsistent with almost all of the rest of the document.

You do not need to _display_ line numbers in order to be able to _highlight_ lines; you can just remove the `linenums="1"` part.

Since the simpler way of selecting namespaces is available, I suppose you will want to use it, removing the need to highlight the (would not exist) Namespace label selectors.